CMakeLists_files.cmake

```diff
@@ -50,6 +50,7 @@ list (APPEND PUBLIC_HEADER_FILES
         opm/flowdiagnostics/ConnectionValues.hpp
         opm/flowdiagnostics/ConnectivityGraph.hpp
         opm/flowdiagnostics/Toolbox.hpp
-        opm/flowdiagnostics/utility/AssembledConnections.hpp
-        opm/flowdiagnostics/utility/RandomVector.hpp
+        opm/utility/graph/tarjan.h
```

Comment:
I think I'd prefer **not** to install `tarjan.h`.  That interface is too low-level for general client consumption.  See commit 3f0ea06.